src/app/locale/getLanguageService.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { LanguageConfig } from '@bigcommerce/checkout-sdk';
+
+import getLanguageService, { initializeLanguageService } from './getLanguageService';
+
+describe('getLanguageService', () => {
+    let languageConfig: LanguageConfig;
+
+    beforeEach(() => {
+        languageConfig = {
+            locale: 'en',
+            locales: {},
+            translations: {},
+            defaultTranslations: {
+                optimized_checkout: {
+                    greeting: 'Hello',
+                },
+            },
+        };
+    });
+
+    it('returns language service if initialized', () => {
+        initializeLanguageService(languageConfig);
+
+        expect(getLanguageService())
+            .toBeDefined();
+        expect(getLanguageService().translate('greeting'))
+            .toEqual('Hello');
+    });
+
+    it('returns language service with fallback translations', () => {
+        initializeLanguageService(languageConfig);
+
+        expect(getLanguageService().translate('address.address_line_1_label'))
+            .toEqual('Address');
+    });
+
+    it('returns language service with override translations', () => {
+        initializeLanguageService({
+            ...languageConfig,
+            locale: 'fr',
+            locales: {
+                'optimized_checkout.greeting': 'fr',
+            },
+            translations: {
+                'optimized_checkout.greeting': 'Bonjour',
+            },
+        });
+
+        expect(getLanguageService().translate('greeting'))
+            .toEqual('Bonjour');
+    });
```

Comment:
I see no test for when the translations are not initialized. I'm assuming that's what `fallbackLanguageService` is for... 😕